fb_instant_articles.module

```diff
@@ -8,6 +8,38 @@
 use Drupal\Core\Routing\RouteMatchInterface;
 
 /**
+ * No ads will automatically be inserted into Instant Articles.
```

Comment:
Can we put these constants in a class somewhere? Same thing with the method below.